src/main/java/com/iota/iri/model/StateDiff.java

```diff
@@ -8,15 +8,31 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Creates a persistable State object, used to map addresses with values in the DB and snapshots.
+ */
 public class StateDiff implements Persistable {
+
+    /** The map storing the address and balance of the current object */
     public Map<Hash, Long> state;
 
+    /**
+     * Returns a byte array of the state map contained in the object. If no data is present in the state,
+     * a new empty byte array is returned instead.
+     */
     public byte[] bytes() {
         return state.entrySet().parallelStream()
                 .map(entry -> ArrayUtils.addAll(entry.getKey().bytes(), Serializer.serialize(entry.getValue())))
                 .reduce(ArrayUtils::addAll)
                 .orElse(new byte[0]);
     }
+
+    /**
+     * Iterates through the given byte array and populates the state map with the contained address
+     * hashes and the associated balance.
+     *
+     * @param bytes The source data to be placed in the State
+     */
     public void read(byte[] bytes) {
```

Comment:
you can remove

Reply:
Since you wrote a specific documentation it is fine imo.
I wouldn't bother to write anything in the first place, but since it is done I won't bother deleting